Tests/UnitTests/Mocks/PaymentComponentDelegateMock.swift

```diff
@@ -12,17 +12,37 @@ final class PaymentComponentDelegateMock: PaymentComponentDelegate {
         onDidSubmit: ((PaymentComponentData, PaymentComponent) -> Void)? = nil,
         onDidFail: ((Error, PaymentComponent) -> Void)? = nil
     ) {
-        self.onDidSubmit = onDidSubmit
-        self.onDidFail = onDidFail
+        self.onDidSubmitClosure = onDidSubmit
+        self.onDidFailClosure = onDidFail
     }
     
-    var onDidSubmit: ((PaymentComponentData, PaymentComponent) -> Void)?
+    // MARK: - onDidSubmit
+
+    var onDidSubmitCallsCount = 0
```

Comment:
`didSubmitCallsCount` and `didSubmitCalled`. the `on` implies it's a closure

Reply:
Yeah, I thought on was part of the method name. Already changed.